code/core/src/manager/components/sidebar/Search.stories.tsx

```diff
@@ -23,77 +22,101 @@ const getLastViewed = () =>
     .filter((item, i) => item.type === 'component' && item.parent && i % 20 === 0)
     .map((component) => ({ storyId: component.id, refId }));
 
+const setQueryParams = fn().mockName('setQueryParams');
+const getQueryParams = fn().mockName('setQueryParams');
+
 const meta = {
   component: Search,
   title: 'Sidebar/Search',
   parameters: { layout: 'fullscreen' },
+  args: {
+    dataset,
+    getLastViewed: (): Selection[] => [],
+    children: () => <SearchResults {...noResults} />,
+  },
```

Comment:
Based on this line https://github.com/storybookjs/storybook/pull/16527/files#diff-29c94008593fc41c988e602ce18ec169d7212632502bff1b0334c2f92dac12d8L44 we probably also want a mock function for `clearLastViewed`

```suggestion
  args: {
    dataset,
    getLastViewed: (): Selection[] => [],
    clearLastViewed: fn().mockName('clearLastViewed'),
    children: () => <SearchResults {...noResults} />,
  },
```

Reply:
the component doesn't have a `clearLastViewed` so this causes a build error. maybe it was in there incorrectly to begin with?

Reply:
aha I see, sounds about right.